web/packages/teleport/src/features.tsx

```diff
@@ -550,7 +550,7 @@ export class FeatureTrust implements TeleportFeature {
   section = ManagementSection.Clusters;
 
   route = {
-    title: 'Trusted Clusters',
+    title: 'Trusted Root Clusters',
```

Comment:
I don't actually know what this field does..

Reply:
React Router uses this to update the header title. it works in a blackbox which is why you cant find its reference anywhere. 
![Screenshot 2024-08-16 at 12 54 11 PM](https://github.com/user-attachments/assets/02473b1f-f72c-4aae-98cb-6fd6c7fda683)